src/test/java/no/unit/nva/PublicationTest.java

```diff
@@ -0,0 +1,111 @@
+package no.unit.nva;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.PublicationStatus;
+import no.unit.nva.model.Reference;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class PublicationTest extends ModelTest {
+
+    ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @DisplayName("Test that each publication type can be round-tripped to and from JSON")
+    @ParameterizedTest(name = "Test that publication with InstanceType {0} can be round-tripped to and from JSON")
+    @CsvSource({
+            "BookAnthology",
+            "BookMonograph",
+            "ChapterArticle",
+            "DegreeBachelor",
+            "DegreeMaster",
+            "DegreePhd",
+            "JournalArticle",
+            "JournalLeader",
+            "JournalLetter",
+            "JournalReview",
+            "JournalShortCommunication",
+            "ReportPolicy",
+            "ReportResearch",
+            "ReportWorkingPaper"
+    })
+    void publicationReturnsValidPublicationWhenInputIsValid(String instanceType) throws Exception {
```

Comment:
Are you testing that the type remains same when serializing/deserializing? E.g. a JournalArticle remains a JournalArticle? Or are you testing as well that the field values are preserved? If you are testing the latter then I would suggest using 'nva-testutils' and the following assertions.

```
    testImplementation group: 'com.github.BIBSYSDEV', name:'nva-testutils', version:'0.1.9'
...
        assertThat(publication,doesNotHaveNullOrEmptyFields());
        assertThat(roundTripped,is(equalTo(publication)));

```